integration/spark/src/main/scala/org/apache/spark/sql/execution/strategy/DDLHelper.scala

```diff
@@ -230,14 +231,22 @@ object DDLHelper {
           .toLowerCase,
         values,
         isColumnRename)
+      var newColumnComment: Option[String] = Option.empty
+      if (changeColumnCommand.newColumn.metadata != null &&
+        changeColumnCommand.newColumn.metadata.contains(CarbonCommonConstants.COLUMN_COMMENT)) {
+        newColumnComment = Some(changeColumnCommand.
+          newColumn.metadata.getString(CarbonCommonConstants.COLUMN_COMMENT))
+      }
```

Comment:
there is already a varibale "newColumn" that can be used above,  or it is better to make "changeColumnCommand.newColumn.metadata" as a variable.

Reply:
I have modified code according to you suggestion.